src/data/customer/__test__/driver.test.js

```diff
@@ -181,65 +195,70 @@ describe('Create new driver ', async () => {
     //   driverTypeIds: [1],
     // };
 
-    const driverInfo = {
-      transactionGroupIds: [180],
-      sendConfirmationSms: false,
-      sendConfirmationEmail: false,
-      driverTypeIds: [2, 3],
-      firstName: 'User',
-      lastName: generateDisplayName(10),
-      email: `${generateDisplayName(10)}@example.com`,
-      password: '123456',
-      birthday: '1980-01-01',
-      phone: `+6592${Math.floor(Math.random() * 1000000)}`,
-      vehicleColor: 'Red',
-      averageSpeed: 60,
-      maximumCapacity: 100,
-      vehicleModelYear: 2018,
-      vehicleLicenseNumber: '12456',
-      vehicleBrand: 'Scooter',
-      vehicleModel: '12456',
-      vehicleTypeId: 1,
-    };
+    it('Test for creating the driver schedule', async () => {
+      const payload = {
+        driverId: driver.id,
+        transactionGroupId: 180,
+        windows: [
+          {
```

Comment:
@vinay3206 should we be testing including the recursion ID and also check if the 'default value' is there?

Reply:
@maartenhemmes there is no recursion or default while creating the driver schedule.
I mean frontend won't be using it with recursion.